synapse/models/orgs.py

```diff
@@ -354,6 +370,40 @@ def getModelDefs(self):
                     }),
                     # TODO a way to encode/normalize requirements.
                 )),
+                ('ou:candidate:method:taxonomy', {}, ()),
+                ('ou:candidate', {}, (
+
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization considering the candidate.'}),
+
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact information of the candidate.'}),
+
+                    ('method', ('ou:candidate:method:taxonomy', {}), {
+                        'doc': 'The method by which the candidate came under consideration.'}),
+
+                    ('submitted', ('time', {}), {
+                        'doc': 'The time the candidate was submitted for consideration.'}),
+
+                    ('note', ('str', {}), {
```

Comment:
Its a little awkward to me that this is a string and not a file:bytes

Reply:
I debated that too. Definitely open to discussion :+1:

Reply:
"Note" feels squishy to me too. The idea of a "cover letter" may be too old school (and dates to the days when  humons actually did initial screenings vs algorithms).  Barring a "please include and upload your cover letter" (a file:bytes) I'm hesitant to extend this to "I dropped you an email" or "I included some arbitrary prose in the 'comments' section of your web page. Which is a long way of saying I'd remove :note until we see a stronger need for it.

Reply:
I think we will get this question almost immediately because the additional "notes" field ( or whatever text input box from a web site where you apply for a job ) will be very useful analytically. Lets discuss today :+1: